features/openshift.feature

```diff
@@ -47,9 +48,29 @@ Feature: Command output from various OpenShift related commands
     """
     OPENSHIFT_URL=https://<ip>:8443
     OPENSHIFT_WEB_CONSOLE=https://<ip>:8443/console
+    DOCKER_REGISTRY=hub.openshift.rhel-cdk.<ip>.xip.io
+    """
+
+    When I successfully run `bundle exec vagrant service-manager env`
```

Comment:
This could probably done as:
```
When I successfully run `bundle exec vagrant service-manager env`
Then stdout from "bundle exec vagrant service-manager env" should contain:
"""
   # your expected output
"""
```